page.tsx

```diff
@@ -88,7 +93,10 @@ export default function createPage(swingsetOptions: SwingsetOptions = {}) {
           </div>
           <Nav navData={filteredNav} />
         </div>
-        <div className={s.stage}>
+        <div className={classNames(s.stage, { [s.isFullscreen]: isFullscreen})}>
+          <div className={s.fullscreenBttn} onClick={handleFullscreenBttnClick}>
+            { isFullscreen ? (<CloseFullscreenIcon />) :  (<FullscreenIcon />)}
+          </div>
```

Comment:
Lets use a button element and add some text content to ensure we have an accessible implementation.